src/app/auth/services/auth.service.ts

```diff
@@ -1,31 +1,59 @@
 import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { User, UserRegistration } from '@shared/models/user.model';
+import { BehaviorSubject, map, Observable } from 'rxjs';
+import { SessionStorageService } from '@app/auth/services/session-storage.service';
+import { Router } from '@angular/router';
+
+interface LoginResponse {
+  result: string;
+}
 
 @Injectable({
   providedIn: 'root',
 })
 export class AuthService {
-  login(user: any) {
-    // replace 'any' with the required interface
+  private API_URL = 'http://localhost:4000/';
```

Comment:
This is a classic example of managing environments in an Angular application. Suppose you are working on the development locally, but it will be deployed to some testing and a production environment later.
Angular has a designated place to manage it - under src/environments folder. You can add your URLs, env variables there individually. Also check the config in .angular file. Read more here: https://dev.to/remshams/replacing-configuration-files-in-angular-builds-based-on-build-targets-28jm